### ISSUE TITLE
[Form/Input/Checkbox/Dropdown] Inconsistent inverted logic

### ISSUE THREAD

**Issue by lubber-de** [fomantic/Fomantic-UI#158]
## Problem
The naming logic of inverted form elements is not consistent.
This could be considered a feature instead of a Bug but currently it's an inconsistent behaviour, which somehow breaks the "inverted" logic for Form Elements:

### Input
has two inverted variants:

`ui inverted input`
![image](https://user-images.githubusercontent.com/18379884/46583796-4f3d0080-ca5c-11e8-865c-f6e45aaddc7e.png)

`ui inverted transparent input`
![image](https://user-images.githubusercontent.com/18379884/46583805-654ac100-ca5c-11e8-9690-520d7df701b2.png)

### Checkbox
has only one variant:
`ui inverted checkbox`
![image](https://user-images.githubusercontent.com/18379884/46583874-1cdfd300-ca5d-11e8-915c-8553ed32f225.png)

The matching "inverted transparent" variant according to input logic is completely missing for checkbox.

### Dropdown 
also has only one variant, But this time "inverted" is completely black without adding "transparent". This breaks the logic acording to input and checkbox
![image](https://user-images.githubusercontent.com/18379884/46583990-61b83980-ca5e-11e8-8026-0675bc7e314a.png)

### Textarea
 does not have any inverted variant at all :disappointed: 

This does not feel consistent and makes clearly inverted forms using the default framework nearly impossible.

## Solution

I suggest implementing the following class additions :

`ui inverted transparent checkout`
Suggested Example:
![image](https://user-images.githubusercontent.com/18379884/46583825-9a571380-ca5c-11e8-8104-816fb16d4f02.png)

`ui inverted transparent dropdown` (which behaves as it currently is without "transparent")
`ui inverted dropdown` (that will be a breaking change)
`ui inverted textarea`
`ui inverted transparent textarea`

## Poll
What do you guys think?

**Comment by prudho** [comment]
Yes, I agree that the `inverted` variation is pretty inconsistent between different components. But for now Fomantic's purpose is to be fully compatible with Semantic, on already existing components, so we're tied with these inconsistencies for now. Maybe for an eventual 3.0 ! 😃
